pkg/scheduler/queue/tenant_queues_test.go

```diff
@@ -196,7 +345,13 @@ func TestQueuesWithQueriers(t *testing.T) {
 	// Add tenant queues.
 	for i := 0; i < numTenants; i++ {
 		uid := TenantID(fmt.Sprintf("tenant-%d", i))
-		getOrAdd(t, qb, uid, maxQueriersPerTenant)
+		//getOrAdd(t, qb, uid, maxQueriersPerTenant)
```

Comment:
comment to delete.
could also rename this test to something like `TestQuerierTenantShuffleShardDistribution`.
in the future PRs we could maybe separate this out as it doesn't actually test any outward behavior about the queues that we rely on, just the tenant-querier assignments.